graphistry/embed_utils.py

```diff
@@ -412,18 +412,33 @@ def predict_links(
         if source is None:
             src = pd.Series(all_nodes)
```

Comment:
maybe add a warning if a cudf is passed in that predict is currently cpu-only?

Reply:
yes,